src/Model/PhasorDynamics/Load/LoadData.hpp

```diff
@@ -6,37 +6,55 @@
  */
 #pragma once
 
-#include <optional>
-#include <string>
+#include <Model/PhasorDynamics/ComponentData.hpp>
 
 namespace GridKit
 {
   namespace PhasorDynamics
   {
+    /// Initial parameters for a load
+    enum class LoadParameters
+    {
+      /// Load resistance
+      R,
+
+      /// Load reactance
+      X,
```

Comment:
Please use compact list of parameters for all components:
```suggestion
      R, ///< Load resistance
      X, ///< Load reactance
```
It is easier to see what parameters are available for each model. In most cases component model developers would recognize the parameter by its name, so a brief comment on the side is sufficient.